widgetssdk/src/main/java/com/glia/widgets/chat/model/history/LinkedChatItem.java

```diff
@@ -0,0 +1,21 @@
+package com.glia.widgets.chat.model.history;
+
+public class LinkedChatItem extends ChatItem {
+
+    private final String messageId;
+    private final long timestamp;
+
+    public LinkedChatItem(String id, int viewType, String messageId, long timestamp) {
+        super(id, viewType);
+        this.messageId = messageId;
+        this.timestamp = timestamp;
+    }
+
+    public String getMessageId() {
+        return messageId;
+    }
+
+    public long getTimestamp() {
+        return timestamp;
+    }
+}
```

Comment:
Hm, I can't understand why `LinkedChatItem` is shown as a change in this PR, taking into account that `LinkedChatItem` is already added to the target branch `feature/MOB-1612_chat_history`. Am I missing anything?